packages/desktop/components/LatestMatchMonitor/index.tsx

```diff
@@ -16,12 +16,19 @@ export const LatestMatchMonitor = () => {
     return <CombatReport combat={latestLocalCombat} matchId={latestLocalCombat.id} viewerIsOwner={true} />;
   }
 
+  const needs470Upgrade = !window.wowarenalogs.obs?.getEncoders;
+
   return (
     <div className="w-full h-full flex flex-col items-center justify-center">
       <div className="hero">
         <div className="hero-content text-center flex flex-col pb-16">
           <h1 className="text-5xl font-bold">Ready for battle</h1>
           <p className="py-6">Please keep WoW Arena Logs running. Your latest match will be reported here.</p>
+          {needs470Upgrade && (
+            <a href="https://wowarenalogs.com/">
```

Comment:
try putting the className onto the <a> tag and get rid of <button> - I think it might be able to do the same thing

Reply:
Actually this needs to be a button not an anchor. That impl will load our page in the electron app :(